viewing_party/party.py

```diff
@@ -1,23 +1,184 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+
+    if not title or not genre or not rating:
+        return None
+    else:
+        movie_dict['title'] = title
+        movie_dict['genre'] = genre
+        movie_dict['rating'] = rating
+    
+    return movie_dict

```

Comment:
Another way of writing this function could be like so: 
```python
def create_movie(title, genre, rating):
    if not title or not genre or not rating:
        return None
    
    return {
        "title": title,
        "genre": genre,
        "rating": rating
    }
   ```
Since all you are doing is creating a dictionary with the arguments your function is given I would think its okay to not use any variables here.